src/core/tables.js

```diff
@@ -0,0 +1,163 @@
+// @ts-check
+// Module core/tables
+// Handles tables in the document.
+// Generates a Table of Tables wherever there is a #tot element.
+
+import {
+  addId,
+  getIntlData,
+  renameElement,
+  showWarning,
+  wrapInner,
+} from "./utils.js";
+import { html } from "./import-maps.js";
+
+export const name = "core/tables";
+
+const localizationStrings = {
+  en: {
+    list_of_tables: "List of Tables",
+    table: "Table ",
+  },
+  ja: {
+    table: "図 ",
+    list_of_tables: "図のリスト",
+  },
+  ko: {
+    table: "그림 ",
+    list_of_tables: "그림 목록",
+  },
+  nl: {
+    table: "Figuur ",
+    list_of_tables: "Lijst met figuren",
+  },
+  es: {
+    table: "Figura ",
+    list_of_tables: "Lista de Figuras",
+  },
+  zh: {
+    table: "图 ",
+    list_of_tables: "规范中包含的图",
+  },
+  de: {
+    table: "Abbildung",
+    list_of_tables: "Abbildungsverzeichnis",
+  },
```

Comment:
Is there a way to get translations for these?
I copied from the "figure" code.

Reply:
For now, we can just keep the English one.